modules/control/control_component.cc

```diff
@@ -157,9 +157,9 @@ Status ControlComponent::ProduceControlCommand(
     ControlCommand *control_command) {
   {
     std::lock_guard<std::mutex> lock(mutex_);
-    local_view_.chassis = latest_chassis_;
-    local_view_.trajectory = latest_trajectory_;
-    local_view_.localization = latest_localization_;
+    local_view_.mutable_chassis()->CopyFrom(latest_chassis_);
+    local_view_.mutable_trajectory()->CopyFrom(latest_trajectory_);
+    local_view_.mutable_localization()->CopyFrom(latest_localization_);
```

Comment:
Just a thought. CyberRT leverages a lot on shared_ptr which allows users to hold a piece of msg without making a copy.